README.md

```diff
@@ -74,36 +74,65 @@ ETH 0x0000000000000000000000000000000000000000 Balance: 2
 
-Redemptions has been published to APM on rinkeby at `redemptions.open.aragonpm.eth`
+The tokens you can add to the Redemptions app are dependent on the blockchain you're on. If you're on the Ethereum mainnet you can add any erc-20 token contract. If you're on a local testnet you can add any token contracts that have been deployed locally. 
 
-To deploy to an organization you can use the [Aragon CLI](https://hack.aragon.org/docs/cli-intro.html).
+Who/what is allowed to control the Redemptions app is determined by the DAO's permissions. You can check the permissions for your DAO in the `Permissions` tab of the Aragon client. Every app currently installed should be list. Click on an app to view it's permissions. 
+- Note: the section you're looking for is "Permissions set on this app", not the permissions available or permissions granted. The "Permissions set on this app" section will tell you who/what can do stuff with the app currently.
 
-```sh
-aragon dao install <dao-address> redemptions.open.aragonpm.eth --app-init-args <vault-address> <token-manager-address>
-```
+<p align="center">
+    <img src="https://raw.githubusercontent.com/1Hive/redemptions-app/master/docs/resources/redemptions-app-permissions.gif" width="600" />
+</p>
 
-The Redemptions app must have the `TRANSFER_ROLE` permission on `Vault` and the `BURN_ROLE` permission on the `Token Manager`.
+### Adding tokens:
 
+The default recommendation is for the Voting app to control adding and removing tokens from the Redemptions app. This way your community can collectively manage the state of Redemptions. Adding tokens to Redemptions via the voting app requires two steps:
+- creating a vote to add the token to the Redeptions app
+- approving the vote to add the token to the Redemptions app
 
-## Using Redemptions
+<p align="center">
+    <img src="https://raw.githubusercontent.com/1Hive/redemptions-app/master/docs/resources/add-token-to-redemptions-app.gif" width="600" />
+</p>
 
-The redemptions app allows organizations to add and remove tokens from a list of eligible tokens. When a user choses to redeem tokens they will receive a proportional share of all eligible tokens in the `Vault`.
+### Removing tokens:
+
+To remove a token from the Redemptions app, open the Redmpetions app in your DAO. Then hover your cursor over the token symbol you would like to remove. You should see the name and amount of the token turn red and display the word "Remove." If you have the Voting app configured to manage token removals (and we recommend this), clicking "Remove" will create a vote to remove the token from Redemptions. Your community will then need to approve this change for it to pass. 
+
+WARNING: if you remove a token from the Redemptions app that has a positive balance, those tokens will be gone. Make sure that you redeem or move all the tokens before taking this action.   
```

Comment:
This should be removed. 

Token balances listed in Redemptions reflect the vault's balance for that token, not the Redemptions contract balance.

Reply:
I think we should still document what the "Remove" button does, but just change it to be accurate. That's what you meant right?

And then to confirm, the "Remove" button on the Redemptions app just removes the token from the redemptionTokenList and thus the UI of the app, but not from the DAO's Vault? Users can add/remove whatever contracts they want from the Redemptions app UI, but what really matters is what's in the Vault

Reply:
Just pushed a fix to the README that explains that the Remove button in the Redemptions UI just removes it from the UI, but not the Vault.